Tests/ContentfulTests/EnvironmentTests.swift

```diff
@@ -72,18 +73,18 @@ class EnvironmentsTests: XCTestCase {
     }
 
     // Copy of tests from SyncTests but using a different environment.
-    func testPerformInitialSync(syncableTypes: SyncSpace.SyncableTypes, action: @escaping (_ space: SyncSpace) -> ()) {
+    func testInitialSyncOnNonMasterEnvironment() {
         let expectation = self.expectation(description: "Sync test expecation")
 
-        EnvironmentsTests.client.sync(syncableTypes: syncableTypes).then { syncSpace in
-
-            fail("Should not successfully sync while connected to non-master nvironment")
-            expectation.fulfill()
-        }.error { error in
-            expect(error).to(beAKindOf(SDKError.self))
+        EnvironmentsTests.client.sync { result in
+            switch result {
+            case .success(let syncSpace):
+                expect(syncSpace.entries.count).to(equal(9))
```

Comment:
Maybe you could check some random entry fields? Maybe like the 1st and the 3rd entry contain a field of `name` of a given value?